less/_input-group.less

```diff
@@ -120,6 +120,7 @@
 	border-top-right-radius: 0;
 	border-bottom-right-radius: 0;
 }
+// this is overriding some border radii and requiring the addition of border-radius: 3px; in Buzz
```

Comment:
Definitely. Doing this task showed me that a lot of cleanup is needed with the inputs.